deflatebench.py

```
@@ -130,22 +130,24 @@ def calculate(results, tempfiles):
-    # Print config info
-
-    print("\n")
+def printinfo():
+    ''' Prints system and configuration info '''
+    print("")
     util.printsysinfo()
+    print("")
     print(f"Tool: {cfgRuns['testtool']} Size: {os.path.getsize(cfgRuns['testtool']):,} B")
     levelrange = f"{cfgRuns['minlevel']}-{cfgRuns['maxlevel']}"
     print(f"Levels: {levelrange:10}")
     print(f"Runs: {str(cfgRuns['runs']):10} Trim worst: {str(cfgRuns['trimworst']):10}")
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)